### PR TITLE
Update link to the Helm project

### PR DESCRIPTION
**Pull request by baijum**
Signed-off-by: Baiju Muthukadan <baiju.m.mail@gmail.com>

### CODE REVIEW

File: content/en/docs/reference/tools/_index.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ to a Kubernetes cluster, troubleshoot them, and manage the cluster and its resou
 
 ## Helm
 
-[`Kubernetes Helm`](https://github.com/kubernetes/helm) is a tool for managing packages of pre-configured
+[`Helm`](https://github.com/helm/helm) is a tool for managing packages of pre-configured
 Kubernetes resources, aka Kubernetes charts.
```
Comment (sftim) [inline content/en/docs/reference/tools/_index.md:30]:
```suggestion
Kubernetes resources. These packages are known as _Helm charts_.
```

File: content/en/docs/reference/tools/_index.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ to a Kubernetes cluster, troubleshoot them, and manage the cluster and its resou
 
 ## Helm
 
-[`Kubernetes Helm`](https://github.com/kubernetes/helm) is a tool for managing packages of pre-configured
+[`Helm`](https://github.com/helm/helm) is a tool for managing packages of pre-configured
```
Comment (sftim) [inline content/en/docs/reference/tools/_index.md:29]:
```suggestion
[Helm](https://helm.sh/) is a tool for managing packages of pre-configured
```

**Review by sftim — COMMENTED** [review]
Thanks. I would link to https://helm.sh/ though

**Comment by baijum** [comment]
@sftim Good suggestions! Committed your changes.

**Pull request merged**